src/hooks/useMediaQuery/index.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  useMediaQuery as useChakraMediaQuery,
+  UseMediaQueryOptions,
+} from '@chakra-ui/react';
+
+import { emBreakpoints as breakpoints } from 'src/themes/shared/breakpoints';
+
+export type QueryProps = {
+  above?: keyof typeof breakpoints;
+  below?: keyof typeof breakpoints;
+};
+
+const useQuery = (props: QueryProps): string => {
+  const toNumber = (em: string) => parseFloat(em);
+
+  if (props.above && props.below) {
+    const min = breakpoints[props.above];
+    const max = breakpoints[props.below];
+    return `(min-width: ${min}) and (max-width: ${toNumber(max) - 0.01}em)`;
+  }
+
+  if (props.above) {
+    const min = breakpoints[props.above];
+    return `(min-width: ${min})`;
+  }
+
+  if (props.below) {
+    const max = breakpoints[props.below];
+    return `(max-width: ${toNumber(max) - 0.01}em)`;
+  }
+
+  throw new Error("Invalid useQuery props: provide 'above' or 'below'");
+};
```

Comment:
If we want to avoid a breaking change (which I think we should in this case), we need to recreate the useQuery function that was exported in Chakra V2. This function allowed us to pass an object of type QueryProps and use our own breakpoints. I believe it's more intuitive to keep using the hook this way.